numba_dppy/compiler.py

```diff
@@ -98,7 +98,7 @@ def compile_with_dppy(pyfunc, return_type, args, debug):
 
     flags = compiler.Flags()
     # Do not compile (generate native code), just lower (to LLVM)
-    if debug:
+    if DEBUGINFO:
```

Comment:
we already have `debug` flag passed to the function. Why are we ignoring it?
Why we are not passing correct flag?

Reply:
This is a temporary solution, because in many functions that call compile_with_dppy no debug is specified at all, as a result of which debug=False
Further study is needed why debug is not indicated in some places and where it should come from

Reply:
I don't think we can accept current implementation

Reply:
@reazulhoque , Can you please tell why "debug" was added to functions parameter and for what purpose?